app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -3,8 +3,8 @@
 public class MatchData {
 
     //Pre-Game
-    int teamNumber = 0;
-    int matchNumber = 0;
+    String teamNumber = "";
```

Comment:
Team and match are always numbers; why making them strings?

Reply:
Has this been resolved?

Reply:
yes